src/borg/archiver/extract_cmd.py

```diff
@@ -38,8 +38,8 @@ def do_extract(self, args, repository, manifest, archive):
         matcher = build_matcher(args.patterns, args.paths)
 
         progress = args.progress
-        output_list = args.output_list
         dry_run = args.dry_run
+        output_list = args.output_list
```

Comment:
as you are back to the original code here, move it 1 line up, so it disappears from the diff.

also, in case you don't do that already, I'ld suggest you read the diff, so you can find such things by yourself.